lib/credentials_manager/cli.rb

```diff
@@ -0,0 +1,107 @@
+require 'credentials_manager/version'
+
+module CredentialsManager
+  class CLI
+    VALID_COMMANDS = %i(add remove)
+
+    REQUIRED_PARAMS = {
+      add: %i(username password),
+      remove: %i(username)
+    }
+
+    def initialize(args)
+      @args = args
+      @options = {}
+
+      begin
+        parse
+        validate
+      rescue => ex
+        # If we have a missing argument, print out the help
+        puts ex.message.capitalize
+        puts parser
+      end
+    end
+
+    # Perform actions based on data
+    def execute
+      case @command
+      when :add
+        CredentialsManager::AccountManager.new(
+          user: @options[:username],
+          password: @options[:password]
+        ).add_to_keychain
+
+        puts "Credential #{@options[:username]}:#{'*' * @options[:password].length} added to keychain."
+      when :remove
+        CredentialsManager::AccountManager.new(
+          user: @options[:username]
+        ).remove_from_keychain
+      end
+    end
+
+    private
+
+    # Generate the options parser
+    def parser
+      @parser ||= OptionParser.new do |opts|
```

Comment:
@KrauseFx Just rewrote this using commander. I've never used this gem before, but honestly, I like it. It seems to be a lot more concise than OptionParser. Why do you want to move off of it?